### DISCUSSION TITLE
Install with Proxmox

### DISCUSSION THREAD

**Discussion by 4x4ff** [Donkie/Spoolman#305] [Q&A] [answered]
Hi,
I am very ignorant when it comes to linux.  Is there a setup guide that I can follow to install this on proxmox?  I am currently running HA on it and would like to add this to it.

**Comment by Tac0T0mmy** [comment]
I just did this, very new to the whole linux proxmox docker thing, I installed alpine linux as its super light weight and then installed docker into that its debian based and works well with proxmox. then follow the docker install on the main page was super simple.

**Comment by TexZeTech** [comment]
What I have done personally.

- setup a vm ([Debian](https://www.debian.org/)) in proxmox
- install docker & portainer CE using the [guide](https://docs.portainer.io/start/install-ce/server/docker/linux) and reference [requirements](https://docs.portainer.io/start/requirements-and-prerequisites)
- go to stacks in portainer
- use the compose example on the main page
- ?????
- PROFIT!!!!

**Comment by Vakrehus** [comment, marked as answer]
Search for "Spoolman" -> https://tteck.github.io/Proxmox/

**Reply by danb35** [reply]
That's actually how I found out about Spoolman, and of course the one-line install command is convenient.
